feature-libs/smartedit/root/http-interceptors/cms-ticket.interceptor.ts

```diff
@@ -11,28 +11,65 @@ import {
   HttpRequest,
 } from '@angular/common/http';
 import { Injectable } from '@angular/core';
+import { PageContext, RoutingService } from '@spartacus/core';
 import { Observable } from 'rxjs';
+import { switchMap, take } from 'rxjs/operators';
 import { SmartEditLauncherService } from '../services/smart-edit-launcher.service';
 
 @Injectable({ providedIn: 'root' })
 export class CmsTicketInterceptor implements HttpInterceptor {
-  constructor(private service: SmartEditLauncherService) {}
+  constructor(
+    private service: SmartEditLauncherService,
+    protected routingService: RoutingService
```

Comment:
You can consider the usage of inject() to avoid breaking changes

Reply:
will do